nova_vm/src/ecmascript/builtins/indexed_collections/typed_array_objects/typed_array_intrinsic_object.rs

```diff
@@ -1852,6 +2033,7 @@ fn search_typed_element<T: Viewable + std::fmt::Debug>(
     search_element: Value,
     k: usize,
     len: usize,
+    ascending: bool,
```

Comment:
Since the internal processing is almost the same, how about adding an option like ascending in find_via_predicate?

Reply:
Make it a `const ASCENDING: bool` generic.

Reply:
thank you! I fixed it! https://github.com/trynova/nova/pull/585/commits/6092afa028efb6af0d13f834cfd1bab69c928a64